src/script/main/index.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Wire
+ * Copyright (C) 2022 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import 'core-js/es7/reflect';
+
+import {Runtime} from '@wireapp/commons';
+import {createRoot} from 'react-dom/client';
+import {container} from 'tsyringe';
+import {enableLogging} from 'Util/LoggerUtil';
+import {loadValue} from 'Util/StorageUtil';
+import {exposeWrapperGlobals} from 'Util/wrapper';
+import {SIGN_OUT_REASON} from '../auth/SignOutReason';
+import {ClientType} from '@wireapp/api-client/src/client/';
+import {Config} from '../Config';
+import {APIClient} from '../service/APIClientSingleton';
+import {Core} from '../service/CoreSingleton';
+import {StorageKey} from '../storage';
+import {App, doRedirect} from './app';
+import {AppContainer} from './components/AppContainer';
+
+$(async () => {
```

Comment:
This is now the index file that does the job of doing all the init side effects.